src/_scss/pages/agencyV2/statusOfFunds/_visualizationSection.scss

```diff
@@ -73,4 +79,5 @@
             }
         }
     }
+    @import "../../../components/roundedtoggle";
```

Comment:
You should be able to remove this import because roundedtoggle.scss is being imported in the above file (ie. src/_scss/pages/agencyV2/statusOfFunds/_statusOfFunds.scss)